src/main/java/com/exasol/adapter/jdbc/BaseColumnMetadataReader.java

```diff
@@ -228,11 +228,12 @@ public DataType mapJdbcType(final JdbcTypeDescription jdbcTypeDescription) {
         case Types.BIT:
         case Types.BOOLEAN:
             return DataType.createBool();
-        case Types.BINARY:
-        case Types.CLOB:
         case Types.TIME:
+            return DataType.createVarChar(100, DataType.ExaCharset.UTF8);
```

Comment:
This `100` was before maximum varchar size, would it make sense to have separate const for time varchar size?

Reply:
Discussed internally.